templates/docs/patterns/equal-height-row.md

```diff
@@ -0,0 +1,99 @@
+---
+wrapper_template: '_layouts/docs.html'
+context:
+  title: Equal height row | Components
+---
+
+The equal height row component aims to provide consistent alignment for grid items within a row format. This is achieved using the CSS `subgrid` feature which allows column grids to share the same layout used in the parent row grid.
+
+This component is an extension of the [grid component](/docs/patterns/grid). There are two ways to use this component:
+
+1. Used as a four column row grid (`.p-equal-height-row`). Each column (`.p-equal-height-row__col`) within the row is a css grid, and may have up to four grid items (`.p-equal-height-row__item`).
+
+2. Nest inside the `25-75` grid pattern on large screen size (`.row--25-75-on-large`). The row (`.p-equal-height-row`) should be placed within the 75% container with a maximum of three columns.
```

Comment:
```suggestion
2. Nested inside the 25/75 grid split pattern on large screen size (`.row--25-75-on-large`). The row (`.p-equal-height-row`) should be placed within the 75% container with a maximum of three columns.
```

This also could be moved down inside the "Three column row" as a description above the example.